css/main.css

```diff
@@ -1,3 +1,225 @@
 /* Add your CSS here */
+*{
 
+}
+
+body{
+    /* display: grid; */
+    /* grid-template-columns: ; */
+    /* min-width: auto; */
+    margin-top: 5rem;
+    font-family: "poppins", sans serif;
+  }
+
+.topnav{
+    display: flex;
+    flex-direction: row;
+     
+}
+
+#myLinks a {
+  text-decoration: none;
+  color: black;
+  float: right;
+  width:7em;
+  margin:10px auto;
+  font-size: 15px;
+}
+
+
+.container{
+  width:100%;
+  display: grid;
+  grid-template-areas: " left right";
+  position:absolute;
+  top: 8rem;
+  padding: 120px 0 0 150px;
+  height: 32em;
+  right: -20em;
+  gap: px;
+}
+
+.main-text{
+    width: 50%;
+    grid-area: right;
+    margin: auto;
+    position: absolute;
+    margin-right: 0px;
+    height: 510px;
+  
+}
+.name-title {
+    display: flex;
+    flex-direction: row;
+    gap: 10px;
+
+}
+.name{
+    margin-top: 20px;
+    height: 20px;
+    font-family: sans-serif;
+    margin-left: 30px;
+}
+.name a{ 
+    line-height: 0.5em;
+    font-size: 20px;
+    margin-left: -20px;
+    
+}
+.title p{
+    font-size: 20px;
+    margin-left: 10px;
+    margin-top: 17px;
+}
+
+
+.blue-button {
+  width: 20px;
+  height: 20px;
+  background-color: #0050FF;
+  margin-top: 20px;
+  margin-left: 14px;
+}
+
+  .grey-box{
```

Comment:
I don't think you need this and I think it's sending you down a rabbit hole of positioning nightmares. Try deleting this entirely and using a linear-gradient background on the body.